app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class Zipcode(db.Model):
+    __tablename__ = 'zip_codes'
+    id = db.Column(db.Integer, primary_key=True)
+    zip_code = db.Column(db.String(10), unique=True)
+    users = db.relationship('User', backref='zip_code', lazy='dynamic')
+    resources = db.relationship('Resource', backref='zip_code', lazy='dynamic')
```

Comment:
I think the location reference to Resource should be in Address, not in ZIPCode? Can this line be moved down to the Address model?